Add pretrained wavernn

yangarbiter opened the pull request.

yangarbiter committed:
Add pretrained wavernn

Commit:
Rename RNNT loss C++ parameters (#1602)

Commit:
Add reduction parameter for RNNT loss (#1590)

vincentqb commented on torchaudio/models/wavernn.py:
```diff
@@ -1,18 +1,29 @@
 
 
+model_urls = {
+    # FIXME Replace URL by final one once determined
```
nit: please remove the comment since the URL has been updated.

vincentqb commented on torchaudio/models/wavernn.py:
```diff
@@ -324,3 +335,21 @@ def forward(self, waveform: Tensor, specgram: Tensor) -> Tensor:
+        pretrained (bool): If True, returns a model pre-trained on LJSpeech
+        progress (bool): If True, displays a progress bar of the download to stderr
+    """
+    kwargs.setdefault("upsample_scales", [5, 5, 11])
+    kwargs.setdefault("n_classes", 30)
+    kwargs.setdefault("n_output", 128)
```
Let's add those as [default](https://github.com/pytorch/audio/blob/master/torchaudio/models/wavernn.py#L230) to the main WaveRNN class instead, with the other defaults.

I'm assuming you took the defaults from [here](https://github.com/pytorch/audio/blob/master/examples/pipeline_wavernn/main.py#L83).

yangarbiter committed:
Add edit_distance

malfet committed:
Update config.[sub|guess] for lame and libmad (#1613)

Needed to allow building on M1

Downloaded from https://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.guess;hb=HEAD and https://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.sub;hb=HEAD

mthrok committed:
Revert CI hot fix (#1614)

Commit:
Expose USE_CUDA in build (#1609)
